cocos/Android.mk

```diff
@@ -284,7 +284,7 @@ LOCAL_WHOLE_STATIC_LIBRARIES += cpufeatures
 
 # define the macro to compile through support/zip_support/ioapi.c
 LOCAL_CFLAGS   :=  -DUSE_FILE32API
-LOCAL_CFLAGS   +=  -fexceptions
+LOCAL_CFLAGS   +=  -fexceptions -fsigned-char
```

Comment:
why add `-fsigned-char` into `Android.mk`, it already been added into `Application.mk`

https://github.com/cocos2d/cocos2d-x/blob/e5b77ad9bfe3356362ff310f33b707dddb00ea8f/templates/cpp-template-default/proj.android/app/jni/Application.mk#L3

Reply:
Thank I will remove it